examples/bpk-component-button/examples.tsx

```diff
@@ -176,6 +177,18 @@ const FullWidthExample = (props: any) => (
   </BpkButtonV2>
 );
 
+const SubmitButtonExample = (props: any) => (
+  <BpkButtonV2 submit {...props}>
+    Submit Button
+  </BpkButtonV2>
+);
+
+const SubmitLoadingButtonExample = (props: any) => (
```

Comment:
ohh this one should probably sit in the bpk-component-loading-button example file 😄